lib/server/controllers/gRPC/GRPCRunController.js

```diff
@@ -34,13 +36,23 @@ class GRPCRunController {
 
     // eslint-disable-next-line require-jsdoc
     async Create(newRunRequest) {
+        // Run can only be updated or created with a valid user
+        if (!await this.userService.validateUserFromGRPCRunRequest(newRunRequest.user_o2_start, newRunRequest.user_o2_stop)) {
```

Comment:
The services have no knowledge of API. gRPC is API, it should not be in services.

The tickets has a mistype, you should not return if there is no user, but simply not update this field

Reply:
Should be fixed now